app/views/search/_sort.html.erb

```diff
@@ -1,11 +1,11 @@
 <div class="dgu-sort">
-  <div class="dgu-sort__apply visuallyhidden">
+  <div class="dgu-sort__apply govuk-visually-hidden">
     <input type="submit" value=<%= t('.accessibility.submit_button') %>/>
   </div>
 
   <div class="dgu-sort__sort-type">
     <label>
-      <span class="visuallyhidden"> <%= t('.accessibility.sort_type_label') %></span>
+      <span class="govuk-visually-hidden"> <%= t('.accessibility.sort_type_label') %></span>
```

Comment:
If you want to visually hide the label, could you put the `govuk-visually-hidden` class on the label itself?